pkg/interface/src/views/landscape/components/Sidebar/SidebarItem.tsx

```diff
@@ -199,22 +191,25 @@ export function SidebarAssociationItem(props: {
     return null;
   }
 
-  const participantNames = (str: string, color: string) => {
+  const participantNames = (str: string) => {
+    const color = isSynced ? (hasUnread || hasNotification) ? 'black' : 'gray' : 'lightGray';
```

Comment:
I don't think the linter allows for nested ternaries like this?

Reply:
Doesn't complain about it here: https://github.com/urbit/urbit/blob/460982d74b72d510bc606477c27764cf7fbc7159/pkg/interface/src/views/landscape/components/Sidebar/SidebarItem.tsx#L39